packages/namekit-react/src/components/DomainMenu.tsx

```diff
@@ -0,0 +1,158 @@
+import { DomainCard } from "@namehash/ens-utils";
+import React, { Fragment, useEffect, useRef, useState } from "react";
+import { DisplayedName } from "./DisplayedName";
+import cc from "classcat";
+import { Link } from "./Link";
+import { OwnerIcon } from "./icons/OwnerIcon";
+import { ThreeVerticalDotsIcon } from "./icons/ThreeVerticalDotsIcon";
+import { BottomDrawer } from "./BottomDrawer";
+import { Transition } from "@headlessui/react";
+
+export const IS_MOBILE_THRESHOLD = 768;
+
+const MenuItem = ({ children }: { children: React.ReactNode }) => (
+  <div className="nk-h-12 md:nk-h-10 nk-w-full nk-flex nk-items-center nk-justify-between md:hover:nk-bg-black md:hover:nk-bg-opacity-5 nk-cursor-pointer nk-transition nk-duration-200">
+    {children}
+  </div>
+);
+
+interface DomainMenuProps {
+  domainCard: DomainCard | null; // when null we show the skeleton
+}
+
+export const DomainMenu = ({ domainCard }: DomainMenuProps) => {
+  const domainMenuRef = useRef<any>();
+  const [menuIsOpen, setMenuIsOpen] = useState(false);
+
+  const [isMobile, setIsMobile] = useState(false);
+  useEffect(() => {
+    window.addEventListener("resize", () => {
+      setIsMobile(
+        window.matchMedia(`(max-width: ${IS_MOBILE_THRESHOLD}px)`).matches,
+      );
+    });
+
+    return () => {
+      window.removeEventListener("resize", () => {
+        setIsMobile(
+          window.matchMedia(`(max-width: ${IS_MOBILE_THRESHOLD}px)`).matches,
+        );
+      });
+    };
+  }, []);
+
+  useEffect(() => {
+    const handleClickOutside = (event: any) => {
+      if (isMobile) return;
+      if (
+        domainMenuRef.current &&
+        !domainMenuRef.current.contains(event.target)
+      ) {
+        setMenuIsOpen(false);
+      }
+    };
+
+    document.addEventListener("mousedown", handleClickOutside);
+    return () => {
+      document.removeEventListener("mousedown", handleClickOutside);
+    };
+  }, [isMobile]);
+
+  if (domainCard === null) return <div className="nk-w-10 nk-h-10" />;
+
+  const textSize = isMobile ? "nk-text-md" : "nk-text-sm";
+  const menuItems = (
+    <>
+      {!!domainCard.ownerAddress && (
+        <MenuItem>
+          <Link
+            target="_blank"
+            variant="secondary"
+            href={`https://app.ens.domains/${domainCard.ownerAddress}`}
```

Comment:
We're open sourcing these components. It's not proper to hardcode things like this. It needs to be something that can be configured dynamically.